### DISCUSSION TITLE
datetime column create error [May 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2212] [Discord Threads]
> 🙋‍♂️ **ofir**  
#### i cheese date and i get this error.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/977861865520627742/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@924381466761265192>, could you please help us with project details? if issue still persistent, do raise an issue here https://github.com/nocodb/nocodb/issues/  
#  
> 🙋‍♂️ **ofir**  
#### Node: **v12.22.6**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **sqlite3**   
#### PackageVersion: **0.90.11**  
#  
> 👨‍🦲 **dstala**  
#### Recommended node version is v14.18.0+. I tried creating a datetime column with below configuration & it worked fine   
####    
#### Node: **v14.18.0**   
#### Arch: **arm64**   
#### Platform: **darwin**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **sqlite3**   
#### PackageVersion: **0.90.11**  
#  
> 🙋‍♂️ **ofir**  
#### I think it's because +03:00  
#  
> 🙋‍♂️ **ofir**  
#### Node: **v16.15.0**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **sqlite3**   
#### PackageVersion: **0.90.11**   
####    
#### again same error  
#  
> 👨‍🦲 **dstala**  
#### Please raise an issue in Github with relevant details. we will look into it.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/977861866166575134)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
